addons/zylann.scatter/tools/plugin.gd

```diff
@@ -182,10 +182,25 @@ func _paint(ray_origin: Vector3, ray_end: Vector3):
 			var instance = _create_pattern_instance()
 			instance.translation = pos
 			instance.rotate_y(rand_range(-PI, PI))
+			
+			var size_range = _palette.get_configured_size_range()
+			var rand_scale = rand_range(size_range[0],size_range[1])/100
```

Comment:
Don't divide by 100 here, the fact it is shown as percentage is purely UI-related. The variable is supposed to be a scale already. Better do this as an implementation detail of `get_configured_size_range`.

Reply:
Sounds good